src/cromshell/utilities/CromshellConfig.py

```diff
@@ -0,0 +1,127 @@
+
+        This function allows users to override the cromwell server set in the
+        cromshell config file using the command line options or environment
+        variables. By default the server URL in the config file is used, however
+        if an URL is specified in the command line or environment then this
+        overrides the default. If a workflow Id is provided without a server URL
+        then the server associated with a workflow id in the submission file
+        overrides the default. If a workflow ID is provided but isn't found
+        in the submission file then the default server is used.
+        """
+
+        if server_user is None and workflow_id is None:
+            cls.LOGGER.info(
+                "Workflow id and cromwell server not specified. Using default"
+            )
+        elif server_user:
+            cls.cromwell_server = server_user
+            CromshellConfig.LOGGER.info(
+                "Cromwell server set to the provided in cli or environment argument."
+            )
+            CromshellConfig.LOGGER.info(cls.cromwell_server)
+        else:
+            with open(cls.submission_file) as f:
+                column = None
+                for line in f:
+                    if workflow_id in line:
+                        column = line.split()
+                if column:
+                    cls.LOGGER.info("Found workflow id in submission file.")
+                    cls.cromwell_server = column[1]
+                    CromshellConfig.LOGGER.info(
+                        "Cromwell server set to matching workflow id in submission file."
+                    )
+                    CromshellConfig.LOGGER.info(cls.cromwell_server)
+                else:
+                    cls.LOGGER.info(
+                        "Unable to find workflow id in submission file. Using default"
+                    )
+
+    @classmethod
+    def user_defined_show_logo(cls, verbosity):
+        """Boolean for Displaying Turtle Logo"""
+
+        if verbosity is None or verbosity < logging.CRITICAL:
+            cls.is_verbose = True
+        else:
+            cls.is_verbose = False
+
+    @staticmethod
+    def __get_config_dir():
+        """Get Path To Cromshell Hidden Directory"""
+
+        config_path = str(Path.home()) + "/.cromshell"
+        Path.mkdir(Path(config_path), exist_ok=True)
+        return config_path
+
+    @staticmethod
+    def __get_submission_file(config_dir):
+        """Get File Path To Cromshell Submission File"""
+
+        submission_file = "{}/all.workflow.database.tsv".format(config_dir)
+        if not Path.exists(Path(submission_file)):
+            Path.touch(Path(submission_file))
+            submission_header = (
+                "DATE{0}CROMWELL_SERVER{0}RUN_ID{0}WDL_NAME{0}STATUS{0}ALIAS".format(
+                    "\t"
+                )
+            )
+            with Path(submission_file).open() as f:
```

Comment:
I was think of using pathlib whenever possible because the [docs](https://docs.python.org/3/library/pathlib.html) mentioned it worked well with different OS